### PR TITLE
Clean up Track 1 mgmt. plane related pipeline settings

### LINKED ISSUE #40721: Wipe out all Track 1 source code

### PR DESCRIPTION
**Pull request by ArthurMa1978**
This step follows up on the changes made in https://github.com/Azure/azure-sdk-for-net/pull/45093.
Fixes https://github.com/Azure/azure-sdk-for-net/issues/40721.

### CODE REVIEW

File: eng/Directory.Build.Common.targets [hunk 1]
```diff
@@ -170,7 +170,7 @@
 
   <ItemGroup Condition="'$(UseProjectReferenceToAzureClients)' == 'true'">
     <!-- Find all non-management projects -->
-    <AzureProjects Include="$(RepoRoot)/sdk/**/src/*.csproj" Exclude="$(RepoRoot)/sdk/**/src/*.Management.*.csproj" ProjectPath="%(Identity)" />
+    <AzureProjects Include="$(RepoRoot)/sdk/**/src/*.csproj" Exclude="$(RepoRoot)/sdk/**/src/Azure.ResourceManager*.csproj" ProjectPath="%(Identity)" />
```
Comment (jsquire) [inline eng/Directory.Build.Common.targets:173]: I don't think this change is what you want.   We should be testing the Resource Manager packages with project references, shouldn't we?

Comment (weshaggard) [reply eng/Directory.Build.Common.targets:173]: I agree we should just remove this Exclude now

**Comment by weshaggard** [comment]
Once this PR goes in we also need to delete the associated pipelines in ADO.

File: eng/Directory.Build.Common.targets (hunk 1 above)
Comment (ArthurMa1978) [reply eng/Directory.Build.Common.targets]: Remove this Exclude

**Comment by ArthurMa1978** [comment]
> Once this PR goes in we also need to delete the associated pipelines in ADO.

We continue to maintain the Track 1 code at this repository and still require the ADO job for hotfixes.

**Review by jsquire — APPROVED** [review]

**Comment by jsquire** [comment]
> We continue to maintain the Track 1 code at this repository https://github.com/Azure/azure-sdk-for-net/tree/mgmt-legacy and still require the ADO job for hotfixes.

But why?   They formally reached end-of-life in September and are no longer published nor supported by Microsoft.    Unsupported means unsupported.

**Review by weshaggard — APPROVED** [review]

**Pull request merged**